src/queues.c

```diff
@@ -617,6 +617,22 @@ struct notification* queues_get_by_id(int id)
         return NULL;
 }
 
+void queues_reapply_all_rules(void)
+{
+        GQueue *recqueues[] = { displayed, waiting, history };
+        for (int i = 0; i < sizeof(recqueues)/sizeof(GQueue*); i++) {
+                for (GList *iter = g_queue_peek_head_link(recqueues[i]); iter;
+                     iter = iter->next) {
+                        struct notification *cur = iter->data;
+                        if (cur->original) {
+                                //rule_print(cur->original);
```

Comment:
Left over from debugging?

Reply:
I left it half intentionally because we might want to give some feedback about what is saved during a reload